x/staking/keeper/delegation.go

```diff
@@ -713,6 +726,8 @@ func (k Keeper) CompleteUnbonding(ctx sdk.Context, delAddr sdk.AccAddress,
 		k.SetUnbondingDelegation(ctx, ubd)
 	}
 
+	k.ExportUnbondingsForAccount(ctx, ubd.DelegatorAddress)
```

Comment:
We are already exporting at end of function `SetUnbondingDelegation`

Better to add exporting in `RemoveUnbondingDelegation`